src/components/lower-info-area/name-build-sub-components/unique-names.vue

```diff
@@ -41,9 +41,11 @@ import { Component, Vue } from 'vue-property-decorator'
 export default class UniqueNames extends Vue {}
 </script>
 
-<style scoped lang="sass">
-  #unique-names
-    p
-      font-size: 14px
-      margin-bottom: 0
+<style scoped lang="scss">
+  #unique-names {
+    p {
+      font-size: 14px;
+      margin-bottom: 0;
```

Comment:
Please remove the margin spec and add "mb-0" to the element's class in the template -- it is easier to have layout specs like this in the template, in case we need to move things around / fix things.

My rule of thumb:
- layout rules in the template
- local styles in the scoped block
- global styles in the assets